src/NUnitEngine/nunit.engine.core/Services/DomainManager.cs

```diff
@@ -296,13 +296,18 @@ public static string GetPrivateBinPath(string basePath, IList<TestPackage> packa
 
         public static string GetPrivateBinPath(string basePath, IList<string> assemblies)
         {
+            Guard.ArgumentNotNull(basePath, nameof(basePath));
+            Guard.ArgumentNotNull(assemblies, nameof(assemblies));
+
+            basePath = Path.GetFullPath(basePath);
+
             List<string> dirList = new List<string>();
             StringBuilder sb = new StringBuilder(200);
 
             foreach( string assembly in assemblies )
             {
                 string dir = PathUtils.RelativePath(
-                    Path.GetFullPath(basePath),
+                    basePath,
                     Path.GetDirectoryName( Path.GetFullPath(assembly) ) );
```

Comment:
What's the reason for the changes in this file, sorry? 🙂

Reply:
The var `basepath` is already a full path before we get here.